src/debug-artifacts.ts

```
@@ -344,9 +344,21 @@ async function createPartialDatabaseBundle(
   if (fs.existsSync(databaseBundlePath)) {
     await del(databaseBundlePath, { force: true });
   }
-  const zip = new AdmZip();
-  zip.addLocalFolder(databasePath);
-  zip.writeZip(databaseBundlePath);
+  const output = fs.createWriteStream(databaseBundlePath);
+  const zip = archiver("zip");
+
+  zip.on("error", (err) => {
+    throw err;
+  });
+
+  zip.on("warning", (err) => {
+    throw err;
+  });
```

Two questions:

1. In what cases will there be a warning and is this really something we want to raise an error on?
2. What will happen if an error is thrown? Where will it be caught and will it be properly handled?

---

Looking at the sample code in the packge, the recommendation is something like this:
```
  if (err.code === "ENOENT") {
    // log warning
  } else {
    // throw error
    throw err;
  }
```

I'll update.